pkg/api/v1/types.go

```diff
@@ -289,3 +290,16 @@ func NewVMReferenceFromName(name string) *VM {
 	vm.SetGroupVersionKind(schema.GroupVersionKind{Group: GroupVersion.Group, Kind: "VM", Version: GroupVersion.Version})
 	return vm
 }
+
+type Migration struct {
+	metav1.TypeMeta         `json:",inline"`
+	ObjectMeta              v1.ObjectMeta     `json:"metadata,omitempty"`
+	MigratingVMName         string            `json:"nodeName,omitempty"`
+	DestinationNodeSelector map[string]string `json:"nodeSelector,omitempty"`
+	StatusMessage           string            `json:"message,omitempty"`
```

Comment:
What is StatusMessage? Is this some kind of Phase indicator?

I would go with `status.Phase` to be consistent